Taskfile.yml

```diff
@@ -20,64 +20,53 @@ tasks:
     desc: Test PostgreSQL
     cmds:
       - task: test-db
-        vars: {DATABASE: postgresql}
+        vars: {TEST_NAME: TestPostgreSQL}
 
   test:mysql:
     desc: Test MySQL
     cmds:
       - task: test-db
-        vars: {DATABASE: mysql}
+        vars: {TEST_NAME: TestMySQL}
 
   test:sqlite:
     desc: Test SQLite
     cmds:
       - task: test-db
-        vars: {DATABASE: sqlite}
+        vars: {TEST_NAME: TestSQLite}
 
   test:sqlserver:
     desc: Test SQLServer
     cmds:
       - task: test-db
-        vars: {DATABASE: sqlserver}
+        vars: {TEST_NAME: TestSQLServer}
 
   test:crdb:
     desc: Test CockroachDB
     cmds:
       - task: test-db
-        vars: {DATABASE: cockroachdb}
+        vars: {TEST_NAME: TestCockroachDB}
 
   test:clickhouse:
     desc: Test Clickhouse
     cmds:
       - task: test-db
-        vars: {DATABASE: clickhouse}
+        vars: {TEST_NAME: TestClickhouse}
 
   test:spanner:
     desc: Test Spanner with GoogleSQL dialect
     cmds:
       - task: test-db
-        vars: {DATABASE: spanner}
+        vars: {TEST_NAME: TestSpanner}
 
   test-db:
     cmds:
-      -  go test work -v -tags {{.DATABASE}} ./...
+      -  go test work -v -run {{.TEST_NAME}} ./...
 
   goreleaser:test:
     desc: Tests release process without publishing
     cmds:
       - goreleaser --snapshot --clean
 
-  docker:
+  test-all:
```

Comment:
We're missing a description (`desc`) on this file. This means that it's not listed on `task --list`.

Also, maybe `test-all` should be replaces with just `test`.

Reply:
Both done